### PR TITLE
Add more logging to load_domain_data script

### PR DESCRIPTION
**Pull request by esoergel**
## Summary
I ran `load_domain_data` and got no feedback for three days before giving up on it.  I'm adding some print statements and a progress bar to give some visibility into the process.

## Safety Assurance

- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] QA labels are set correctly
- [x] I am certain that this PR will not introduce a regression for the reasons below

### Automated test coverage

None

### QA Plan

N/A

### Safety story

This change only affects a management command that's run only on blank environments to populate them with data.  A full QA pass and data integrity checks should be carried out at the end of that process.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/apps/dump_reload/couch/load.py [hunk 1]
```diff
@@ -26,43 +26,41 @@ class CouchDataLoader(DataLoader):
     def __init__(self, object_filter=None, stdout=None, stderr=None):
         super().__init__(object_filter, stdout, stderr)
         self._dbs = {}
-        self.success_counter = Counter()
-
-    def _get_db_for_doc_type(self, doc_type):
```
Comment (esoergel) [inline corehq/apps/dump_reload/couch/load.py:31]: I moved this below and didn't otherwise change it

File: corehq/apps/dump_reload/interface.py [hunk 2]
```diff
@@ -69,3 +73,10 @@ def load_from_file(self, extracted_dump_path, force=False):
             )
 
         return total_object_count, loaded_object_count
+
+
+def _get_gzfile_line_count(file_path):
+    # This has to iterate through the whole file, which takes time (~15 minutes
+    # for an 11G sql.gz file), but it's worth it to predict completion time
```
Comment (esoergel) [inline corehq/apps/dump_reload/interface.py:80]: I'd be interested to hear other options for this

Comment (kaapstorm) [reply corehq/apps/dump_reload/interface.py:80]: What you've done here seems to be the cleanest way to do this.

You probably thought of using
```python
    result = subprocess.check_output(f'zcat {filepath} | wc -l')
    return int(result.decode('utf-8'))
```
I'm not sure how much faster that might be. `zcat` and `wc -l` might be faster than using Python. (I tested on a small compressed log file, and `zcat` + `wc -l` was a little bit faster.) Firing up a subprocess would be an initial cost, but would be outweighed by the 11G. But your code is much neater than this.

(And of course although you could get the uncompressed size of the SQL quickly from the gzip metadata, it won't tell you how many objects it contains.)

**Review by kaapstorm — APPROVED** [review]
A progress bar is a great idea!

I must remember to use it more often.

File: corehq/apps/dump_reload/interface.py (hunk 2 above)
Comment (esoergel) [reply corehq/apps/dump_reload/interface.py:80]: I did try running
```bash
$ time zcat filename | wc -l
```
on the commandline vs this implementation, and it was faster - something like 12 vs 15 minutes.  Since it wasn't an order of magnitude improvement, I thought simplicity wins out, but I don't feel too strongly about it.

Comment (snopoke) [reply corehq/apps/dump_reload/interface.py:80]: what if we changed the dump command to include some metadata like total number of docs?

Comment (esoergel) [reply corehq/apps/dump_reload/interface.py:80]: That could work - the dump output is a zip of a few different files, so it doesn't sound hard to have a meta file with some json data or something.  I'll look in to that.
